store/search.ts

```diff
@@ -0,0 +1,25 @@
+import { ActionTree, MutationTree } from 'vuex';
+
+export interface SearchState {
+  query: string;
+}
+
+export const state = (): SearchState => ({
+  query: ''
+});
```

Comment:
```suggestion
const defaultState = (): SearchState => ({
  query: ''
});
export const state = defaultState;
```

And please add RESET_SEARCH commit and resetSearch action to call `Object.assign(state, defaultState())`

Reply:
@MrTimscampi If you wish I can add it for you :)

Reply:
Added